spark/src/test/java/org/apache/iceberg/spark/source/TestSparkSchema.java

```diff
@@ -46,12 +46,12 @@
 
 public abstract class TestSparkSchema {
 
-  private static final Configuration CONF = new Configuration();
-  private static final Schema SCHEMA = new Schema(
+  protected static final Configuration CONF = new Configuration();
+  protected static final Schema SCHEMA = new Schema(
       optional(1, "id", Types.IntegerType.get()),
       optional(2, "data", Types.StringType.get())
   );
-  private static SparkSession spark = null;
+  protected static SparkSession spark = null;
```

Comment:
This doesn't need to change since the tests should be in the `spark` module instead of in a subclass of `TestSparkSchema`.

Reply:
ack